custom_components/tuya_local/devices/klarta_stor2_purifier.yaml

```diff
@@ -0,0 +1,149 @@
+name: Klarta Stor 2 air purifier
+products:
+  - id: ku6ae3y8ad6tbx3b
+    name: "KLARTA STOR 2 "
+primary_entity:
+  entity: switch
+  icon: "mdi:air-purifier"
+  dps:
+    - id: 1
+      type: boolean
+      name: switch
+secondary_entities:
+  - entity: sensor
+    class: pm25
+    dps:
+      - id: 2
+        type: integer
+        name: sensor
+        class: measurement
+        unit: ugm3
+  - entity: select
+    name: Mode
+    dps:
+      - id: 3
+        name: option
+        type: string
+        mapping:
+          - dps_val: auto
+            value: "Auto"
+          - dps_val: "manual"
+            value: "Manual"
+          - dps_val: "sleep"
+            value: "Sleep"
+          - dps_val: "screenoff"
+            value: "Turn off LEDs"
+  - entity: select
+    name: Speed
+    dps:
+      - id: 4
+        name: option
+        type: string
+        icon: "mdi:fan"
+        mapping:
+          - dps_val: speedmode1
+            value: "Silent"
+          - dps_val: speedmode2
+            value: "Silent+"
+          - dps_val: speedmode3
+            value: "Slow"
+          - dps_val: speedmode4
+            value: "Slow+"
+          - dps_val: speedmode5
+            value: "Fast"
+          - dps_val: speedmode6
+            value: "Turbo"
+  - entity: sensor
+    name: Active filter life
+    icon: "mdi:air-filter"
+    category: diagnostic
+    dps:
+      - id: 5
+        type: integer
+        name: sensor
+        unit: "%"
+  - entity: select
+    name: AutoMode
```

Comment:
Should be the human readable name, following [HA naming conventions](https://developers.home-assistant.io/blog/2022/07/10/entity_naming/), so "Auto mode".